docs/rules/html-closing-bracket-new-line.md

```diff
@@ -0,0 +1,86 @@
+---
+pageClass: 'rule-details'
+sidebarDepth: 0
+title: 'svelte/html-closing-bracket-new-line'
+description: "Require or disallow a line break before tag's closing brackets"
+---
+
+# svelte/html-closing-bracket-new-line
+
+> Require or disallow a line break before tag's closing brackets
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> **_This rule has not been released yet._** </badge>
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+This rule enforces a line break (or no line break) before tag's closing brackets, which can also be configured to be enforced on self-closing tags.
+
+<ESLintCodeBlock fix>
+
+<!--eslint-skip-->
+
+```svelte
+<script>
+  /* eslint svelte/brackets-same-line: "error" */
+</script>
+
+<!-- ✓ GOOD -->
+<div></div>
+<div
+  multiline
+>
+  Children
+</div>
+
+<SelfClosing />
+<SelfClosing
+  multiline
+/>
+
+<!-- ✗ BAD -->
+
+<div
+></div>
+<div
+  multiline>
+  Children
+</div>
+
+<SelfClosing
+/>
+<SelfClosing
+  multiline/>
+```
+
+</ESLintCodeBlock>
+
+## :wrench: Options
+
+```jsonc
```

Comment:
@ota-meshi, `jsonc` feels like the right lang to use, does it have any issue with the docs syntax highlighting?

Reply:
I think the syntax highlighting works fine on our website 👍